requirements-base.txt

```diff
@@ -27,7 +27,7 @@ mmh3>=2.3.1,<2.4
 parsimonious==0.8.0
 petname>=2.6,<2.7
 phonenumberslite>=8.11.0,<8.12.0
-Pillow>=6.2.2,<7.0.0
+Pillow>=8.0.0,<9.0.0  # Pillow 8.x supports Python 3.6 to 3.9
```

Comment:
I wonder if it makes sense to keep the lower bound temporarily until `getsentry`'s exact version lands first.

Reply:
No, you should do them at the same time (as close as possible). Land getsentry right after sentry lands.